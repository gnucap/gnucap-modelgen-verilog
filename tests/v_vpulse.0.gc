
attach ../vams/vpulse.so

verilog

vpulse #(.val0(0), .val1(1)) v1(in, 0);

list

//print dc v(nodes)
//dc basic
print tran v(nodes) v(v1) control(0) iter(0)
tran 10 basic

status notime
