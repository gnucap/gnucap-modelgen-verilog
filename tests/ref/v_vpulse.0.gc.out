vpulse #(.dc(.5),.val0(0),.val1(1)) v1 (.p(in),.n(0));
#           v(in)     
 0.         0.5       
#Time       v(in)      v(v1)     
 0.         1.         1.        
 10.1E-9    1.         1.        
 1.         1.         0.        
 1.         0.         0.        
 2.         0.         1.        
 2.         1.         1.        
 3.         1.         0.        
 3.         0.         0.        
 4.         0.         1.        
 4.         1.         1.        
 5.         1.         0.        
 5.         0.         0.        
 6.         0.         1.        
 6.         1.         1.        
 7.         1.         0.        
 7.         0.         0.        
 8.         0.         1.        
 8.         1.         1.        
 9.         1.         0.        
 9.         0.         0.        
 10.        0.         1.        
Gnucap   System status
iterations: op=0, dc=3, tran=63, fourier=0, total=87
transient timesteps: accepted=21, rejected=0, total=21
nodes: user=1, subckt=0, model=0, total=1
dctran density=100.0%, ac density=100.0%
